### PR TITLE
Framework review - Optimize moveos_stdlib code style

### PR DESCRIPTION
**Pull request by pause125**
## Summary

Part of #921 

Optimize moveos_stdlib code style.

### CODE REVIEW

File: moveos/moveos-stdlib/moveos-stdlib/sources/context.move [hunk 1]
```diff
@@ -1,7 +1,9 @@
+// Copyright (c) RoochNetwork
```
Comment (baichuan3) [inline moveos/moveos-stdlib/moveos-stdlib/sources/context.move:1]: Maybe can consider updating the framework license checking and adding logic to `scripts/update_licence.sh`, to ensure that subsequent frameworks have licenses.

Comment (jolestar) [reply moveos/moveos-stdlib/moveos-stdlib/sources/context.move:1]: The `update_licence` skip the move file #386. 

If we add the license to the Move code, we can enable the check.

**Review by jolestar — APPROVED** [review]

File: moveos/moveos-stdlib/moveos-stdlib/sources/context.move (hunk 1 above)
Comment (jolestar) [reply moveos/moveos-stdlib/moveos-stdlib/sources/context.move:1]: The `update_licence` skip Move files #386. If we decide to add the license to Move files, we can enable it.

**Comment by jolestar** [comment]
I merged this PR first because this PR change all Move files.

**Pull request merged**